lua/nvim-lsp-installer/servers/rust_analyzer/init.lua

```diff
@@ -7,6 +7,8 @@ local Data = require "nvim-lsp-installer.data"
 
 local coalesce, when = Data.coalesce, Data.when
 
+local libc = platform.get_libc()
```

Comment:
This seems unused?

Reply:
Seems I forgot to use that further down in that same file to avoid lookups. I'll make a modification here in a bit.